tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_child_device_measurement/run.py

```diff
@@ -0,0 +1,47 @@
+from environment_c8y import EnvironmentC8y
+
+import time
+import json
+import os
+
+
+class TedgeMapperC8yChildDevice(EnvironmentC8y):
+    def setup(self):
+        super().setup()
+        child_device_fragment = self.cumulocity.get_child_device_of_thin_edge_device_by_name(
+            self.project.device, "thin-edge-child")
+        self.assertThat('actual == expected',
+                        actual=child_device_fragment, expected=None)
+        self.addCleanupFunction(self.test_cleanup)
+
+    def execute(self):
+        self.startProcess(
+            command=self.tedge,
+            arguments=["mqtt", "pub",
+                       "tedge/measurements/thin-edge-child",
+                       '{"temperature": 12, "time": "2021-06-15T17:01:15.806+02:00"}'],
+            stdouterr="tedge_pub",
+            environs=os.environ
+        )
+
+        # Waiting for the mapped measurement message to reach the Cloud
+        time.sleep(1)
+
+        self.child_device_fragment = self.cumulocity.get_child_device_of_thin_edge_device_by_name(
+            self.project.device, "thin-edge-child")
+
```

Comment:
I would like to add one more publishing measurement with the different temperature value and timestamp, and check if that second measurement is created on the same child device. The aim is to ensure that the mapper doesn't always create a new device with a known child id.